### ISSUE TITLE
New release with fixed GET_ALL_PAGES_IN_DIR

### ISSUE THREAD

**Issue by excavador** [pottava/aws-s3-proxy#32]
Any chance to see new release with this fix?
https://github.com/pottava/aws-s3-proxy/pull/31

**Comment by pottava** [comment]
@excavador Could you try the `pottava/s3-proxy:2.0rc` docker image? Or you can build with `make build` command with the latest source code.

**Comment by excavador** [comment]
@pottava 
1. `pottava/s3-proxy:2.0rc` made before PR merge.
2. I already has own version, but I prefer to switch to vanilla one.

**Comment by pottava** [comment]
@excavador 

I just released `pottava/s3-proxy:2.0`. I am glad if you have time to try!

**Comment by jeremydonahue** [comment]
@pottava Thanks for creating the v2 release, however I get an error when I try to use it with `go get` while building a custom image:

```
$ go get -d -u github.com/pottava/aws-s3-proxy@v2.0.1
go: finding github.com v2.0.1
go: finding github.com/pottava/aws-s3-proxy v2.0.1
go: finding github.com/pottava v2.0.1
go: finding github.com/pottava/aws-s3-proxy v2.0.1
go get github.com/pottava/aws-s3-proxy@v2.0.1: github.com/pottava/aws-s3-proxy@v2.0.1: invalid version: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v2
```
The previous version works fine with `go get` (`v1.4.1`). It seems to me that to fix this, Go wants you to modify the module name in `go.mod` like so:

```
- module github.com/pottava/aws-s3-proxy
+ module github.com/pottava/aws-s3-proxy/v2
```
...which is dumb, but `¯\_(ツ)_/¯`
